org.cvfosammmm.Lemma.yml

```diff
@@ -0,0 +1,49 @@
+id: org.cvfosammmm.Lemma
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: lemma
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=host
+  - --env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux
+  - --env=LC_ALL=C
+  - --talk-name=ca.desrt.dconf
+  - --env=DCONF_USER_CONFIG_DIR=.config/dconf
+modules:
+  - name: python3-pyxdg
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "pyxdg" --no-build-isolation
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/e5/8d/cf41b66a8110670e3ad03dab9b759704eeed07fa96e90fdc0357b2ba70e2/pyxdg-0.28-py2.py3-none-any.whl
+        sha256: bdaf595999a0178ecea4052b7f4195569c1ff4d344567bccdc12dfdf02d545ab
+  - name: python3-uharfbuzz
+    buildsystem: simple
+    build-options:
+        build-args:
+          - --share=network
```

Comment:
this is not allowed. build must be offline.

Reply:
I couldn't manage to build it without this arg. Any ideas?

Reply:
any dependencies should be installed
we have some tooling for that https://github.com/flatpak/flatpak-builder-tools

But there is no information you provided to help you further.

This apply to other modules too.

Reply:
The pip tool doesn't work.

This is the output for the fontconfig module:
`========================================================================
Downloading sources
========================================================================
Running: "pip3 download --exists-action=i --dest /tmp/pip-generator-python3-fontconfig-pyur0_0i8o -r /tmp/requirements.ahv2vw41"
DEPRECATION: Loading egg at /usr/local/lib/python3.11/dist-packages/Python_FreeType-0.62-py3.11.egg is deprecated. pip 24.3 will enforce this behaviour change. A possible replacement is to use pip for package installation.. Discussion can be found at https://github.com/pypa/pip/issues/12330
Collecting fontconfig-py (from -r /tmp/requirements.ahv2vw41 (line 1))
  Using cached fontconfig_py-0.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (1.4 kB)
Using cached fontconfig_py-0.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.3 MB)
Saved /tmp/pip-generator-python3-fontconfig-pyur0_0i8o/fontconfig_py-0.1.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl
Successfully downloaded fontconfig-py
========================================================================
Downloading arch independent packages
========================================================================
Traceback (most recent call last):
  File "/home/hp/Setzer/flatpak/pip/./flatpak-pip-generator", line 291, in <module>
    url = get_tar_package_url_pypi(name, version)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/hp/Setzer/flatpak/pip/./flatpak-pip-generator", line 91, in get_tar_package_url_pypi
    raise Exception(err)
Exception: Failed to get fontconfig_py-0.1.2 source from https://pypi.org/pypi/fontconfig_py/0.1.2/json
`

Reply:
No, the harfbuzz build fails, see https://buildbot.flathub.org/#/builders/26/builds/12298

The manifest was generated with `flatpak-pip-generator`.

Reply:
I remove the modules that couldn't be built.

Reply:
what were they needed for then?

Reply:
I have alternatives to the libraries now directly in the Lemma repo. That's why I could remove them.